File: java/250322_B2_1100.java
/**
 * https://www.acmicpc.net/problem/1100
 */
import java.io.*;

class Main {
	public static void main(String[] args) throws IOException {
		try (var br = new BufferedReader(new InputStreamReader(System.in))) {
			int sum = 0;
			for (int i = 0; i < 8; i++) {
				var inp = br.readLine().toCharArray();
				for (int j = i % 2; j < 8; j += 2) {
					if (inp[j] == 'F') {
						sum++;
					}
				}
			}
			System.out.println(sum);
		}
	}
}

File: rust/src/bin/250322_B2_1100.rs
/**
 * https://www.acmicpc.net/problem/1100
 */
use { std::io };

fn main() {
    let mut sum: usize = 0;
    for i in 0..8 {
        let inp: String = read_input();
        for j in (0..8).step_by(2) {
            let chk = (i % 2) + j;
            if inp.chars().nth(chk) == Some('F') {
                sum += 1;
            }
        }
    }
    println!("{}", sum);
}

fn read_input() -> String {
    let mut inp = String::new();
    io::stdin().read_line(&mut inp).unwrap();
    inp
}
